docs/crds/Listener.md

```diff
@@ -185,4 +221,150 @@ The following table explains the various fields in the `Listener.action.redirect
+
+## Listener.policies.httpprofile
+
+This attribute represents the HTTP configuration for the front-end CS virtual server.
+
+Following is an example for the `Listener.policies.httpprofil` attribute.
+```yml
+    policies:
+     httpprofile:
+      config:
+       websocket: "ENABLED"
+
+    policies:
+     httpprofile:
+     preconfigured: test-http-profile
+```
+The following table explains the various fields in the `Listener.policies.httpprofile` attribute.
+
+| Field         | Description                                             | Type          | Required |
+|---------------|---------------------------------------------------------|---------------|----------|
+| `preconfigured`      | Specifies the name of the preconfigured HTTP profile that is to be used for the front end CS virtual server. This profile must be present in the Citrix ADC before applying the policy. Otherwise, Listener resource fails to apply. Either `preconfigured` or `config` is required.      | String        | No      |
+| `config`     | Specifies the [HTTP](https://developer-docs.citrix.com/projects/citrix-adc-nitro-api-reference/en/latest/configuration/ns/nshttpprofile/) profile settings for the front-end virtual server. You can specify the key-value pair as shown in the example to tune the HTTP protocol characteristics of the virtual server.  | Object        | No      |
+
+## Listener.policies.sslprofile
+
+This attribute represents the SSL profile for the front-end CS virtual server. 
+
+Following is an example for the `Listener.policies.httpprofil` attribute.
+```yml
+    policies:
+     sslprofile:
+      config:
+       ssl3: "ENABLED"
+
+    policies:
+     sslprofile:
+      preconfigured: test-ssl-profile
+```
+The following table explains the various fields in the `Listener.policies.sslprofile` attribute.
+
+
+| Field         | Description                                             | Type          | Required |
+|---------------|---------------------------------------------------------|---------------|----------|
+| `preconfigured`      | Specifies the name of the preconfigured SSL profile that is to be used for the front-end SSL virtual server. This profile must be present in the Citrix ADC before applying the policy. Otherwise, the Listener resource fails to apply. Either `preconfigured` or `config` is required.    | String        | No      |
+| `config`     | Specifies the SSL policy configuration for the front-end virtual server. You can specify the key-value pair as shown in the example to tune the [SSL](https://docs.citrix.com/en-us/citrix-adc/current-release/ssl/ssl-profiles/ssl-enabling-the-default-profile.html) characteristics of the virtual server. **Note:** You must enable the default profiles using `set ssl parameter -defaultProfile ENABLED` in Citrix ADC for using the advanced SSL features.  | Object        | No      |
```

Comment:
Specifies the SSL profile configuration.  ( SSL profile must link to https://developer-docs.citrix.com/projects/citrix-adc-nitro-api-reference/en/latest/configuration/ssl/sslprofile/)